vip-scanner/class-base-check.php

```diff
@@ -55,11 +55,23 @@ protected function get_all_files( $files ) {
 
 	protected function filter_files( $files, $type = '' ) {
 		$files = (array) $files;
-		if( $type ) {
-			if( isset( $files[$type] ) )
-				return $files[$type];
-			else
-				return array();
+		if ( $type ) {
+			if ( is_array( $type ) ) {
+				$file_types = array();
```

Comment:
The naming of this variable is confusing, as we're not storing an array of file types, but rather files themselves.